spec/unit/cli/list_spec.rb

```diff
@@ -43,6 +48,13 @@
       expect { command }.to output(/\n\nTotal: 3$/).to_stdout_from_any_process
     end
 
+    it 'prints a subset of the resource details when the attribute option is given' do
```

Comment:
how about adding context here for rest of the test cases: context 'with filter' do